### ISSUE TITLE
Add sources to datapoints

### ISSUE THREAD

**Issue by MansMeg** [swerik-project/riksdagen-politicians#3]
To simplify understanding the varying data quality, information and references to all individual data points in the repository need to be maintained and added.

A first suggestion has been to add a source column to all csv file in the catalogue. Although this has the problem that we will not be able to give a reference to a separate element. It also does not easily match with wikidata structures with references for individual datapoints.

**Suggested solution:**
We introduce a table (csv) called source with five columns:
- file: file to reference
- row_id: row to reference
- column_name: column to reference (if it is NULL, the whole row is referenced)
- source: a bibtex reference
- source_pages: specific page(s) if relevant

**Comment by BobBorges** [comment]
A problem with both the solutions above is that it could be potentially quite difficult to keep in sync with updates from wikidata, since our csv files don't necessarily have a distinct row index. 

I want to add another possibility to the thread for discussion -- no sources on the metadata files that are dynamically updated from wikidata. Instead we add a source column only to static files that are verified info used for data integrity / quality assessment tests (like the known_MPs_catalog). In this way, we don't create a mess of partially referenced data in every file that we need to somehow keep synchronized with dynamically updated metadata files, rather we have static files with a reference in each row that we can check against any updates from wikidata. 

Lastly, just a reminder that we already have the bibtex catalog set up.

**Comment by ljo** [comment]
I can help with creating examples for the different cases so we can have a clearer overview.

**Comment by MansMeg** [comment]
Great @ljo !

I think we should also think a little, based on Joakim's comment, that we might want to have a more general reference system, not just one to point to elements in a table/matrix file.

**Comment by ljo** [comment]
@MansMeg Yes, I agree it should be a more general reference system. The use cases are unfortunately not ready for today's meeting.
